src/components/PageHeadingFlows.vue

```diff
@@ -1,19 +1,8 @@
 <template>
-  <PageHeading class="page-heading-flows" :crumbs="crumbs">
-    <template v-if="can.access.collections" #after-crumbs>
-      <p-button :to="routes.flowCollections()" secondary>
-        Add from Collection
-      </p-button>
-    </template>
-  </PageHeading>
```

Comment:
This link isn't being curated atm so we've decided to remove navigation to it for the time being; easy enough to add later should things change with that project.

Reply:
In favor of this. It was way to prominent and didn't seem to provide much value. But should we add a menu and add it there? Just so we retain the functionality.

Reply:
Chatted with bill about this - for now, no, but we'll retain the components and routes so we don't break any perm links